cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/component/ClouderaManagerDefaultStackRepoDetails.java

```diff
@@ -31,9 +31,9 @@ public void setCdhVersion(String cdhVersion) {
 
     @Override
     public String toString() {
-        return new StringJoiner(", ", ClouderaManagerDefaultStackRepoDetails.class.getSimpleName() + "[", "]")
+        return new StringJoiner(", ", ClouderaManagerDefaultStackRepoDetails.class.getSimpleName() + '[', "]")
```

Comment:
The next string is convertible to char as well

Reply:
`char` is not `CharacterSequence` which is the type of the input parameter of `StringJoiner`. `String` + `char` is a `CharacterSequence` as well